black-mulberry-data-transfer/src/main/java/com/black/mulberry/data/transfer/response/UserAuthResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.black.mulberry.data.transfer.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class UserAuthResponse {
+    private UserResponse user;
```

Comment:
why we created UserAuthResponse when its include the same data that includes in UserResponse? please check this

Reply:
We left this to add the jwt token in the next task, that's why I returned it like this․
If you think i did something wrong please share